src/main/java/io/reactivex/Flowable.java

```diff
@@ -10288,6 +10289,100 @@ public final Completable ignoreElements() {
     }
 
     /**
+     * Parallelizes the flow by creating multiple 'rails' (equal to the number of CPUs)
+     * and dispatches the upstream items to them in a round-robin fashion.
+     * <p>
+     * Note that the rails don't execute in parallel on their own and one needs to
+     * apply {@link ParallelFlowable#runOn(Scheduler)} to specify the Scheduler where
```

Comment:
It is the same logic as with regular factory methods such as `just`, `range`, `fromIterable` don't take a `Scheduler`, plus you can apply multiple `runOn`'s on a sequence at different stages. For example create a pipeline with stages of parallelism=2 and 3 stages in total.